critiquebrainz/frontend/static/styles/main.less

```diff
@@ -490,3 +490,22 @@ a#edit-review { margin-top: 20px; }
     margin-top: 25px;
   }
 }
+
+// Moderator page
+
+#list {
+  margin-top: 30px;
+}
+#list img {
```

Comment:
We already have a class for an avatar, which can be used here. https://github.com/metabrainz/critiquebrainz/blob/master/critiquebrainz/frontend/static/styles/main.less#L64-L70